src/psyclone/psyir/nodes/operation.py

```diff
@@ -105,6 +106,15 @@ def node_str(self, colour=True):
         return self.coloured_name(colour) + \
             "[operator:'" + self._operator.name + "']"
 
+    def is_elemental(self):
+        '''
+        :returns: whether this operation is elemental (provided with an input \
+            array it will apply the operation individually to each of the \
+            array elements and return and array with the results).
+        :rtype: bool
+        '''
+        return self.operator.name not in self._non_elemental_names
```

Comment:
As mentioned below, I think this would be nicer if we kept a list of the operators themselves rather than their names.

Reply:
Done